src/event_loop.rs

```diff
@@ -86,9 +86,29 @@ pub enum ControlFlow {
     /// arrives or the given time is reached.
     WaitUntil(Instant),
     /// Send a `LoopDestroyed` event and stop the event loop. This variant is *sticky* - once set,
-    /// `control_flow` cannot be changed from `Exit`, and any future attempts to do so will result
-    /// in the `control_flow` parameter being reset to `Exit`.
-    Exit,
+    /// `control_flow` cannot be changed from `ExitWithCode`, and any future attempts to do so will
+    /// result in the `control_flow` parameter being reset to `ExitWithCode`.
+    ///
+    /// The contained number will be used as exit code. The [`Exit`] constant is a shortcut for this
+    /// with exit code 0.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **Android**, **iOS**, **WASM**: The supplied exit code is unused.
+    /// - **Unix-alikes** (**Wayland** or **X11**): On some platforms, only the 8 least significant
```

Comment:
```suggestion
    /// - **Unix**: On most Unix-like platforms, only the 8 least significant
```

I think sticking close to `std`'s documentation is best here.